### PR TITLE
Tickets/sitcom 689

### PR DESCRIPTION
**Pull request by patrickingraham**

### CODE REVIEW

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+**Announcement Protocol:**
+
+- Announcements are required for all work that may affect the summit control network or its infrastructure.
+  This includes Nublado updates, invasive networking changes, or interactions with any CSC.
+- Utilize threads whenever possible to minimize cross-talk and/or interference between simultaneous discussions
+
+**Announcement Guidelines and Procedure:**
+
+- Make the announcement when ready to begin work.
+  Include and estimate of how long the systems will be in use.
```
Comment (edennihy) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:113]: "an estimate"

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  Include name of other channel where conversation on the work will take place (if appropriate).
```
Comment (edennihy) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:114]: Include the names of other channels.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+  List the CSC(s) you plan on using and a 1 or 2-sentence synopsis of what you are doing.
+
+- Announce when work is completed.
+  This should not be buried in a previous thread but be a new message
```
Comment (edennihy) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:118]: "but instead be a new message"

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+
+Communications in this room should be minimal.
+This is a single channel for both telescopes which is actively monitored by telescope specialists and anyone controlling the system.
+For maximum efficiency, primary communication should be via Zoom and not in the channel.
+Breakout rooms can be used if multiple simultaneous conversations are required.
+
+This channel is meant to be used as the area where the copying/pasting of code snippets happens while supporting via zoom, therefore context will often not be provided.
+Troubleshooting in this area should be either system-wide, or issues which benefit from having multiple people see and understand the diagnosis.
+
+An example of troubleshooting belonging to this channel is found in `this Slack snippet <_static/dialog_example.png>`_.
```
Comment (edennihy) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:148]: This was a broken link for me, not sure why.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+
+#summit-maintel and #summit-auxtel
+----------------------------------
+
+These rooms are for telescope system specific asynchronous conversation with remote support.
+They are passively monitored by Observing Specialists and there is to be no expectation of an immediate response.
```
Comment (edennihy) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:155]: You should add that they are passively monitored by Observing Specialists _on shift_. Also I think you can drop "to be"

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+Generally, the content will be read and start and end of a shift, or during quiet periods.
+
+These channels are actively monitored by shift manager(s), who will directly contact with Observing Specialists if required.
```
Comment (edennihy) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:158]: typo: directly contact Observing Specialists

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+
+Example content:
+
+- Discussion of faults that were already recovered.
+  For example, "Experienced issue with shutter again, reported on OBS-12345."
+
+- Announcements such as, “Calibrations completed in filters XYZ” for people may be interested to know but do not follow #summit-announce.
```
Comment (edennihy) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:165]: typo: for people who may be interested but do not follow #summit-announce.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+#auxtel-operations and #maintel-operations
+------------------------------------------
+
+These channels are also telescope system specific but are dedicated for communication between people providing external support, such as looking at data as it comes in.
+The external supporters are not directly tied in to on-site activities and are often not aware of the immediate status of the observatory.
+
+These channels are passively monitored by the shift managers, who elevate concerns/issues when required.
+Observing Specialists will often be participating as well, but are not required.
+On-shift Observing Specialists should not be @-'d at any time. 
+If required, contact the shift manager and they will take the issue to the real-time communication channel(s) if and when appropriate.
```
Comment (edennihy) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:179]: I think you should consider dropping this part, it's just too inviting and we already say above the shift manager will elevate issues.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+  List the CSC(s) you plan on using and a 1 or 2-sentence synopsis of what you are doing.
+
+- Announce when work is completed.
+  This should not be buried in a previous thread but be a new message
+  
+- Announcements for testing using other test-stands are to be made on test-stand specific channels, following the same rules.
+  This includes #rubinobs-tucson-teststand and #rubinobs-base-teststand
+
+
+.. examples include:
```
Comment (edennihy) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:124]: Any reason you left the example content commented out here?

File: Daytime-Nighttime-Interactions/_static/Slack-channel-diagram.png
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/_static/Slack-channel-diagram.png:1]: How about "Not invited to contribute content to the channel" ?

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+.. :target: ../../../Daytime-Nighttime-Interactions/_static/OBS_Jira_workflow.png
+..     :target: _static/OBS_Jira_workflow.png
+
+Final states
+------------
+The final states are the ultimate destination of a reported ticket.
+When a ticket reaches this state, is it considered to be resolved and will no longer be actively managed by the Fault Handling Management process.
+We start by explaining these states because they are referenced throughout the remainder of the document, specifically when describing state transitions.
+
+- ``Done``: All work required to address the issue has been completed.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst:67]: Not to my knowledge. There might be custom things you can define, but I'd rather not go down that path...

[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst:67]: Note that using the double ticks doesn't use italics though, it changes the font type (making it look like code).
We have done this in other places when describing the state machines etc.

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst [hunk 11]
```diff
@@ -0,0 +1,164 @@
...
+There are multiple ways that tickets reach these states, which are explained in the sections below.
+The ``Cancelled`` and ``Duplicate`` states are special because they can be transitioned to from any active state in the workflow.
+
+There is one transition out of these states, which is from the ``Done`` state to the ``Reported`` state.
+This transition should be rare. 
+It is reserved for cases when a job is in a completed state, but either the exact error occurs again, or someone deems it is not satisfactorily completed. 
+The ticket is then transitioned to ``Reported`` so it can be triaged and prioritized accordingly by a Triage manager.
+
+
+Active states
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst:85]: I actually did this originally and then removed it.
I have mixed feelings about it because in some ways I don't know that they are really sections as then they go into tables of contents etc.
I think the nail in the coffin was actually linked to the highlighting issue, if you change it to a section, then you can link the section, which is good, but you can't do the text highlighting, which is bad.

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst [hunk 12]
```diff
@@ -0,0 +1,164 @@
...
+-------------
+This section describes each of the active states as well as any transition from that state to another.
+
+- ``Reported:`` This is the default state that a ticket enters when it is initially filed. 
+  The ticket should be assigned to the Triage Manager on-shift.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst:90]: Good catch on the conflict, but I think it's incorrect here.
Observers won't know who the Triage manager is (or at least aren't obliged to know), so I thought we said that it goes unassigned, and then the Triage manager assigns it and transitions it to "To Do," no ?

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst [hunk 13]
```diff
@@ -0,0 +1,164 @@
...
+  The ticket remains in this state until resources have been allocated and work is initiated.
+
+  At this stage, if the initial assignee determines they are unable to complete the work, they must perform the following:
+
+  - Find the appropriate assignee.
+    If they are unable to do so, then the assignee should be the current Triage Manager.
+  - Verify that the new assignee is able to work on the ticket
+  - Post a comment on the ticket indicating the reason for the re-assignment.
+
+    - *Transition to* ``In Progress``: After initial assignee assessment and once a work plan has been identified and resources allocated to perform the work, the ticket should be transitioned to ``In Progress``.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst:112]: Wow. that took me awhile to see! I moved them out a tab until we converge on the above.

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst [hunk 14]
```diff
@@ -0,0 +1,164 @@
...
+
+    - *Transition to* ``Done``: If the work needed to implement a solution can be rapidly completed, it can transition directly to ``Done``. 
+      This transition is reserved for minor issues requiring no testing/tracking or urgent tickets requiring rapid response.
+
+    - *Transition to* ``Testing``: Work has been done to address the issue and a potential fix has been identified and implemented. 
+      If the fix is deployed and additional data must be collected before deeming success, it can be transitioned to Testing.
+
+- ``Tracking``: This state is reserved for issues which have been considered by the assignee and deemed to have insufficient information to determine the root cause or implement a solution. 
+  If a particular data collection procedure must be followed it must be detailed in the comments section and either the Triage Manger or Run Manager should be informed so that the procedure can be communicated to the relevant stakeholders, specifically the operators.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst:131]: but it's anyone operating that night right?
Could be Bruno, you, me... whomever.
I changed to "night crews" -- that work?

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst [hunk 15]
```diff
@@ -0,0 +1,164 @@
...
+
+    - *Transition to* ``Done``: If the fix that was deployed is considered successful, upon completion of the work it can be transitioned directly to Done. A short description of the final issue resolution should be included in the comments section of the ticket before transitioning to ``Done``.
+
+
+
+OBS Ticket Management Process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The management process for the OBS project is still being developed and enacted, therefore this section will be updated as the process matures.
+Generally, the ticket management strategy is expected to consist of a rotation of Triage Managers, combined with a regularly scheduled meeting of the Triage managers to revisit tickets that are stalled, stale, or are not receiving sufficient attention.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst:153]: I don't know if that's practical as that is ~12 people... it should certainly include the observing specialist manager though. That job is in the hiring process so how about adding that?

File: Daytime-Nighttime-Interactions/fault-reporting.rst [hunk 16]
```diff
@@ -39,11 +41,11 @@ Some guidelines to keep in mind are:
 
 - Facts first. 
   The author of the fault report should provide as many details as possible, including screenshots, telescope telemetry, and timestamps for future investigation.
-- If the reporter is unsure of who to assign the ticket to, leave it unassigned and alert the day time staff for further triage.
+- Leave the ticket unassigned, it will be triaged in the morning or self-assigned.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/fault-reporting.rst:44]: acknowledged, but not resolving or giving a 👍 until we converge.

File: Daytime-Nighttime-Interactions/fault-reporting.rst [hunk 17]
```diff
@@ -111,9 +113,9 @@ Guidelines For Calculating Time Loss
 .. It is preferred to include them as a bulleted or enumerated list.
 .. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
 
-- If the problem can be troubleshooted while taking images on sky, or proceeding with another task, that time won’t count to fault loss.
+- If the problem can be troubleshooted while taking images on sky, or proceeding with another task, that time will not count towards a fault loss.
 - If the problem happens before or after 12 degree twilight, there is no need to account for fault loss.
-    - As soon as science time begins however, the clock starts ticking.
+    - As soon as science time begins however, the time loss begins accumulating.
 - If the problem happens during bad weather, or no observing is taking place, there is no need to deduct for time loss.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/fault-reporting.rst:119]: good point, but doesn't "science time" only include potential on-sky time? I made a change, but you should check you agree.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 18]
```diff
@@ -0,0 +1,196 @@
...
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#################################################
+Slack Channel Communication with Summit Personnel
+#################################################
+
+The LSSTC Slack space is used by hundreds of people with varying levels of interest in summit operations. 
+The goal of this document is to communicate the acceptable usage of the specific set of channels that are focused on nighttime operations.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:28]: changed, but slightly different than the proposal

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst (hunk 5 above)
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:155]: added on-shift, but modified the phrase a bit to remove the "to be"

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst (hunk 4 above)
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:148]: Ya, I hit this the other day with another doc. I don't know why it fails. I pinged jonathan on #rubinobs-sitcom-docs

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst (hunk 9 above)
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:124]: I didn't really think they were required, plus I thought if they are required they belong somewhere else... yet i felt bad for removing them.
I just wiped them though.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 19]
```diff
@@ -0,0 +1,196 @@
...
+People who participate in testing or infrastructure support/development are expected to be actively monitoring this channel.
+
+Because work is being coordinated across multiple channels, the creation of this channel aims to restrict the need to make announcements multiple places and avoid the scenario of accidentally not informing all impacted parties.
+
+For items having substantial impact, scheduling should first be being performed as part of the `SUMMIT Jira calendar <https://jira.lsstcorp.org/secure/DoItBetterCalendar.jspa>`_ process.
+
+**Announcement Protocol:**
+
+- Announcements are required for all work that may affect the summit control network or its infrastructure.
+  This includes Nublado updates, invasive networking changes, or interactions with any CSC.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:107]: Re-worded to address the theme of the comment, but didn't use the exact wording.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 20]
```diff
@@ -0,0 +1,196 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section provides additional details regarding the use of each channel.
+Note that all channels require active monitoring by everyone to ensure the conversation stays inline with the channel's expected content.
+The shift managers are the primary disciplinarians but observing specialists and others should police too!
+
+#summit-announce
+----------------
+This channel is used to announce activities that may impact other summit users.
+Announcements on this nor any other channel does *NOT* grant permission from personnel on the ground regarding whether or not it is safe to perform the test.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:95]: annoying text formatting in rst again... same issue for the states etc. I switched to bold though... does that help?

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 21]
```diff
@@ -0,0 +1,196 @@
...
+
+Any summit activities and/or testing, including anything utilizing the network, requires **two** announcements: one when starting, the other when finished, to prevent people from stepping on one another.
+People who participate in testing or infrastructure support/development are expected to be actively monitoring this channel.
+
+Because work is being coordinated across multiple channels, the creation of this channel aims to restrict the need to make announcements multiple places and avoid the scenario of accidentally not informing all impacted parties.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:100]: I'm confused, what is the claim we can't make?
So this text is actually from the original rules of the channel that are bookmarked at the top!
So it's not changing the rules at all...
I did remove the "creation of this" bit though.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 22]
```diff
@@ -0,0 +1,196 @@
...
+To better communicate the expectations for each of the roles, and demonstrating the information flow, the list of required Slack channels have been categorized into Tiers. 
+We use the "summit" prefix for channels that are tightly coupled to summit operations.
+
+- **Tier 0:** Real-time information that the applicable roles are always monitoring and are expected to immediately react to (when on shift and interacting with summit systems). 
+  Responding to a message may result in a loss of sky time, which is acceptable. 
+  Communication in this area meant to be extremely curt. 
+  People providing support are encouraged to do so via Zoom and not via Slack
+
+- **Tier 1:** Used for asynchronous information communication regarding the state of summit systems. 
+  These channels are telescope specific, do not require immediate response, and are passively monitored by Observing Specialists.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:69]: changed to observers as well.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 23]
```diff
@@ -0,0 +1,196 @@
...
+- Off-site Summit Support: Personnel that call in to assist or perform summit-based activities but remotely (e.g. from the base or Tucson)
+
+- Project Personnel: General project members that have slack accounts
+
+- External Contributors: People that share the same slack user-space but do not participate to operations  (e.g. DESC).
+
+People in each of these roles will have varying responsibilities regarding which information they need to be paying attention to and/or responding to.
+The ultimate goal is to facilitate effective communication, but minimize noise to the observers that will minimize efficiency.
+It is recognized that all project personnel are capable of contributing useful input, however, unless the people are very close to the on-going tasks there exists a significant awareness gap.
+Therefore, a flow via which anyone on the project can participate has been developed that passes through the key personnel which can address the project member's concerns and/or questions.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:58]: Added, but changed slightly.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 24]
```diff
@@ -0,0 +1,196 @@
...
+- Summit Personnel: These are people that are actively working on the summit. This category includes the Triage Manager (future role), even if they are not physically located on the summit.
+
+- Off-site Summit Support: Personnel that call in to assist or perform summit-based activities but remotely (e.g. from the base or Tucson)
+
+- Project Personnel: General project members that have slack accounts
+
+- External Contributors: People that share the same slack user-space but do not participate to operations  (e.g. DESC).
+
+People in each of these roles will have varying responsibilities regarding which information they need to be paying attention to and/or responding to.
+The ultimate goal is to facilitate effective communication, but minimize noise to the observers that will minimize efficiency.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:56]: Kept the theme of your comment but modified the text a bit.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 25]
```diff
@@ -0,0 +1,196 @@
...
+   
+   Slack is not meant to be the primary source of information for summit observing support.
+   When remote support personnel are working directly with summit personnel, direct communication is strongly advised (e.g. Zoom).
+
+To derive the layout of the new channels, several key aspects were discussed and/or included. 
+For ease of presentation, it was decided to classify the stakeholders by the following roles:
+
+- Observing Specialists: Nominally the primary operators of observatory systems, however, for the sake of this document this category includes anyone actively participating in system operations. 
+
+- Shift Managers: Organizer(s) of the current observing run that perform coordination of observations, relay information between the day and night crews (beyond what is to be included in the end-of-day report), and provide support during the night and day.
```
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:44]: Sorry, not sure which comment you're referring to...
I agree that it'd be better if the shift manager role was defined elsewhere, but for the moment it isn't (it actually is on a confluence page somewhere but that only adds confusion as it's an HR definition).
My hope is that once we define this elsewhere we can modify it here but for the moment I think we're stuck?

File: Daytime-Nighttime-Interactions/fault-reporting.rst
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/fault-reporting.rst:1]: That's fair, let's poll for opinions after releasing the page.

File: Daytime-Nighttime-Interactions/_static/Slack-channel-diagram.png
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/_static/Slack-channel-diagram.png:1]: That's a good middle ground. Revised version here: https://docs.google.com/document/d/1tc90IAY1AW4g-3PapQw2uhrcxhOIflWobL7SbXapPLQ/edit?usp=sharing

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst (hunk 11 above)
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst:85]: That's a bummer, okay we can stick with the formatting as-is for now.

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst (hunk 13 above)
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst]: I think if you move the transition descriptions above L105 then we won't have the issue of two bulleted lists at different levels on top of each other.

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst (hunk 14 above)
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst]: Good point, I thought it was just a misnomer. Night crews works.

File: Daytime-Nighttime-Interactions/fault-reporting.rst (hunk 16 above)
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/fault-reporting.rst:44]: Resolved with your solution above.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst (hunk 25 above)
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:44]: Okay, that's fine to leave it here. I don't really have a better solution either.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst (hunk 21 above)
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst]: Thanks, I think it really was just the "creation of this" part that threw me off. Reading the revised version I don't see any issues.

File: Daytime-Nighttime-Interactions/fault-reporting.rst (hunk 17 above)
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/fault-reporting.rst:119]: The change doesn't quite capture what the issue I had, which is just with the beginning of L119. The distinction of faults that happen during "bad weather" and "good weather" is needless. Can we intsead reword L119 to say "If a fault occurs while we are closed due to bad weather, the time lost should be reported as 0." Then we can move L118 below this phrase to capture both cases of how to count time when either 12 deg twilight starts or observing starts after weather clears up.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst (hunk 4 above)
Comment (edennihy) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst]: Resolved by dropping the image in-line with the text.

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst (hunk 15 above)
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst:155]: Adding observing specialist manager works for me.

File: Daytime-Nighttime-Interactions/fault-handling-workflow.rst (hunk 13 above)
[the remark this answers is not in the document]
Comment (edennihy) [reply Daytime-Nighttime-Interactions/fault-handling-workflow.rst]: I think if you move the transition descriptions above the text that starts in L105 then we won't have the issue of two bulleted lists at different levels on top of each other.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst (hunk 8 above)
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/slack-channel-usage.rst:175]: oops, forgot to address this one, sorry.
My qualm here is that the channel rules will be linked from each channel individually (to these headings). So we need to repeat information at some level as I don't think people will actually read the entire thing (sadly).
On another note, we'll also need to link the schedule of shift managers... or people will just @channel and annoy everyone.

File: Daytime-Nighttime-Interactions/_static/Slack-channel-diagram.png
[the remark this answers is not in the document]
Comment (patrickingraham) [reply Daytime-Nighttime-Interactions/_static/Slack-channel-diagram.png:1]: Figure swapped.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 26]
```diff
@@ -0,0 +1,192 @@
...
+- Shift Managers: Organizer(s) of the current observing run that perform coordination of observations, relay information between the day and night crews (beyond what is to be included in the end-of-day report), and provide support during the night and day. 
+  They also help to address questions, comments, and filter information between the broader community and summit personnel.
+
+- Summit Personnel: These are people that are actively working on the summit. This category includes the Triage Manager (future role), even if they are not physically located on the summit.
+
+- Off-site Summit Support: Personnel that call in to assist or perform summit-based activities but remotely (e.g. from the base or Tucson)
+
+- Project Personnel: General project members that have slack accounts
+
+- External Contributors: People that share the same slack user-space but do not participate to operations  (e.g. DESC).
```
Comment (tribeiro) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:53]: Is it "not participate to operations" or "not participate in operations"?

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 27]
```diff
@@ -0,0 +1,192 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section provides additional details regarding the use of each channel.
+Note that all channels require active monitoring by everyone to ensure the conversation stays inline with the channel's expected content.
+The shift managers are the primary disciplinarians but observers and others should police too!
+
+#summit-announce
+----------------
+This channel is used to announce activities that may impact other summit users.
+Announcements on this nor any other channel does **NOT** grant permission from personnel on the ground regarding whether or not it is safe to perform the test.
```
Comment (tribeiro) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:96]: This phrase is a bit confusing. I think the "nor" is giving the impression of a double negative. My non-native speaker brain is trying to rephrase it like this:

> Announcements in this, or in any other channel, does **NOT** grant permission from personnel on the ground regarding whether or not it is safe to perform the test.

I am afraid non-native speakers will have a hard time understanding the meaning of this phrase. Maybe rephrase it?

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 28]
```diff
@@ -0,0 +1,192 @@
...
+**Announcement Protocol:**
+
+- Announcements are required for all work that may affect the summit control network or its infrastructure.
+  This includes hardware testing, CSC updates/deployments, configuration changes, Nublado updates, invasive networking changes and any on-site activities being performed by summit personnel.
+- Utilize threads whenever possible to minimize cross-talk and/or interference between simultaneous discussions
+
+**Announcement Guidelines and Procedure:**
+
+- Make the announcement when ready to begin work.
+  Include an estimate of how long the systems will be in use.
```
Comment (tribeiro) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:114]: Maybe itemize these?

```
- Make the announcement when ready to begin work.

   - (NEW) Keep the announcement short (1-paragraph).
     If additional information is required, use threads.
   - Include an estimate of how long the systems will be in use.
   - Include the names of other channels where conversation on the work will take place (if appropriate).
   - List the CSC(s) you plan on using and a 1 or 2-sentence synopsis of what you are doing.
```

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 29]
```diff
@@ -0,0 +1,192 @@
...
+  Include the names of other channels where conversation on the work will take place (if appropriate).
+  List the CSC(s) you plan on using and a 1 or 2-sentence synopsis of what you are doing.
+
+- Announce when work is completed.
+  This should not be buried in a previous thread but instead should be a new message.
```
Comment (tribeiro) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:119]: We have mostly been using threads to announce when the work is completed. I think this works better than having a separate message saying the work was completed as it links start/finish to the same message. Maybe we should use the "Also send to ..." feature? but only if there are other announcements after the original one?

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 30]
```diff
@@ -0,0 +1,192 @@
...
+  
+- Announcements for testing using other test-stands are to be made on test-stand specific channels, following the same rules.
```
Comment (tribeiro) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:121]: Maybe start with a more enforcing phrase? Something like:

```
This channel is dedicated to summit operations.
Announcements for testing using other test-stands...
```

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 31]
```diff
@@ -0,0 +1,192 @@
...
+
+    An example of the type of troubleshooting conversation that should take place in #summit-control-room.
+
+
+#summit-maintel and #summit-auxtel
+----------------------------------
+
+These rooms are for telescope system specific asynchronous conversation with remote support.
+They are passively monitored by on-shift observers with no expectation of an immediate response.
+Generally, the content will be read and start and end of a shift, or during quiet periods.
```
Comment (tribeiro) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:152]: Not sure I understood this phrase. Did you mean:

```
Generally, the content will be read at the start and end of a shift, or during quiet periods.
```

e.g.: "... will be read and start..." -> "... will be read at start..."

**Review by tribeiro — COMMENTED** [review]
Parachuting in this PR without invitation... :)

File: Daytime-Nighttime-Interactions/_static/Slack-channel-diagram.png
Comment (edennihy) [inline Daytime-Nighttime-Interactions/_static/Slack-channel-diagram.png:1]: I realized after the fact that we also needed to update the Observing Specialists role to Observers to be consistent with the descriptions above. Link to updated figure -> https://docs.google.com/document/d/1tc90IAY1AW4g-3PapQw2uhrcxhOIflWobL7SbXapPLQ/edit

**Review by edennihy — APPROVED** [review]
Thanks for addressing the remaining items in the second round of changes, approved. Please see my comment on the updated slack figure to grab one final change before merging.
